server/src/main/java/org/jboss/as/server/logging/ServerLogger.java

```diff
@@ -1345,6 +1345,17 @@
     @Message(id = 277, value = "Failed to load SSH Credentials %s")
     RuntimeException failedToLoadSSHCredentials(@Cause Throwable cause, String message);
 
+    @LogMessage(level = INFO)
+    @Message(id = 278, value = "The configuration history is managed through Git")
```

Comment:
would it make sense to mention the path to the  cloned Git repository in that INFO log?

Reply:
I need to move on from this issue now, the logging I added was to get to the bottom of a test failure - I think other messages will help identify the path if needed.